sdcm/cluster.py

```diff
@@ -3361,6 +3361,14 @@ def get_nodetool_info(node):
         info_res = yaml.load(proper_yaml_output)
         return info_res
 
+    def get_node_host_id(self, node):
+        address = node.ip_address
+        full_nodetool_status = self.get_nodetool_status()
```

Comment:
you should pass the node here, otherwise it randomly select a node, if you call it once one of the nodes are down
it might fail